docs/tox.ini

```diff
@@ -18,19 +18,22 @@ commands =
 
 [testenv:docs]
 description = Build the Sphinx docs
+allowlist_externals = rm
 deps = pip-tools
 commands_pre =
     pip-sync {toxinidir}/requirements.txt
 commands =
-    sphinx-build -W --keep-going . _build/html
+    rm -rf __pycache__ _build/
+    sphinx-build -W --keep-going -b dirhtml . _build/html
+    python -m http.server -b 127.0.0.1 -d _build/html/
 
 [testenv:serve]
 description = Live reload and serve the Sphinx docs
 deps = pip-tools
 commands_pre =
     pip-sync {toxinidir}/requirements.txt
 commands =
-    sphinx-autobuild . _build/html
+    sphinx-autobuild -b dirhtml . _build/html
```

Comment:
If we're incorporating "serve" into `tox -e docs`, can we get rid of `tox -e serve`?